test/util.ts

```diff
@@ -1,4 +1,6 @@
 import { FunctionDecl } from "../src";
+import { synthesizeEventPattern } from "../src/eventbridge/eventpattern";
+import { FnLsEventPattern } from "../src/eventbridge/eventpattern/types";
```

Comment:
What does `FnLs` mean?

Reply:
Functionless, need a better acronym and give up and use the full word...

Reply:
Renamed